qiskit_ibm_runtime/batch.py

```diff
@@ -21,14 +21,76 @@
 
 
 class Batch(Session):
-    """Class for creating a batch mode in Qiskit Runtime."""
+    """Class for creating a batch mode in Qiskit Runtime.
+
+    Just like ``session``, a Qiskit Runtime ``batch`` allows you to group a collection of
+    iterative calls to the quantum computer. Batch mode can shorten processing time if all jobs
+    can be provided at the outset. If you want to submit iterative jobs, use sessions instead.
+
+    Using batch mode has these benefits:
+        - The jobs' classical computation, such as compilation, is run in parallel.
+          Thus, running multiple jobs in a batch is significantly faster than running them serially.
+
+        - There is no delay between job, which can help avoid drift.
+
+    You can open a Qiskit Runtime batch using this ``Batch`` class and submit jobs
+    to one or more primitives.
+
+    For example::
+
+        from qiskit.circuit import QuantumCircuit, QuantumRegister, ClassicalRegister
+        from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+        from qiskit_ibm_runtime import Batch, SamplerV2 as Sampler
+
+        service = QiskitRuntimeService()
+        backend = service.least_busy(operational=True, simulator=False)
+
+        # Bell Circuit
+        qr = QuantumRegister(2, name="qr")
+        cr = ClassicalRegister(2, name="cr")
+        qc = QuantumCircuit(qr, cr, name="bell")
+        qc.h(qr[0])
+        qc.cx(qr[0], qr[1])
+        qc.measure(qr, cr)
+
+        pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+        isa_circuit = pm.run(qc)
+
+        with Batch(backend=backend) as batch:
+            sampler = Sampler(batch)
+            job = sampler.run([isa_circuit])
+            pub_result = job.result()[0]
+            print(f"Sampler job ID: {job.job_id()}")
+            print(f"Counts: {pub_result.data.cr.get_counts()}")
+
+    """
 
     def __init__(
         self,
         service: Optional[QiskitRuntimeService] = None,
         backend: Optional[Union[str, BackendV1, BackendV2]] = None,
         max_time: Optional[Union[int, str]] = None,
     ):
+        """Batch constructor.
+
+        Args:
+            service: Optional instance of the ``QiskitRuntimeService`` class.
+                If ``None``, the service associated with the backend, if known, is used.
+                Otherwise ``QiskitRuntimeService()`` is used to initialize
+                your default saved account.
+            backend: Optional instance of ``Backend`` class or string name of backend.
+                If not specified, a backend will be selected automatically (IBM Cloud channel only).
```

Comment:
```suggestion
```

Reply:
Because of the ISA circuit thing, you have to specify a backend on Cloud too now.